controllers/adminController.js

```diff
@@ -6,12 +6,12 @@ function viewAdmin(req, res) {
   if (!res.locals.isAdmin) {
     return res.status(403).render("403")
   }
-  res.render("admin")
+  res.render("admin/admin")
```

Comment:
When splitting out into directories, it's standard practice to name the default view "index.ejs"

Reply:
File name changed to index